src/app/lib/styledGrid/index.jsx

```diff
@@ -27,6 +29,160 @@ import {
   gridContainerLargeCss,
 } from '../layoutGrid';
 
+export const MediumContentContainer = styled.div`
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    margin: 0 auto;
+    max-width: 46.4rem;
+  }
+`;
+
+const defaultColumns = {
+  group0: 6,
+  group1: 6,
+  group2: 6,
+  group3: 6,
+  group4: 6,
+  group5: 12,
+};
+
+const mediumColumns = {
+  ...defaultColumns,
+  group3: 5,
+  group4: 5,
+  group5: 10,
+};
+
+export const MediumGridWrapper = ({ children }) => {
+  return (
+    <Grid
+      enableGelGutters
+      parentColumns={defaultColumns}
+      columns={defaultColumns}
+    >
+      {children}
+    </Grid>
+  );
+};
+
+MediumGridWrapper.propTypes = {
+  children: node.isRequired,
+};
+
+export const GridItem = ({ children }) => {
```

Comment:
`GridItem` seems very similar to `GridItemMedium` other than the margin configurations. It would be nice to see on the PR examples of `GridItem`'s use so I can better understand where they are intended to be used.

Reply:
They are similar. I guess this could be made to one function and pass in a prop like `enableMargin`

Reply:
Done here https://github.com/bbc/simorgh/pull/6618/commits/6af6c4897a2eeec30281946d8edc5f2795adc4fd. `GridItem` is used on the `Paragraph` and `Image` container.